homeassistant/components/notify/clicksend.py

```diff
@@ -29,7 +29,7 @@ def validate_sender(config):
     """Set the optional sender name if sender name is not provided."""
     if CONF_SENDER in config:
         return config
-    config[CONF_SENDER] = config[CONF_RECIPIENT]
+    config[CONF_SENDER] = 'hass'
```

Comment:
'hass' should be extracted as a constant

Reply:
Would you want this added to the homeassistant.const as I cannot really see any place for it in there and that seems to be the only extracted constants file I can see.

Reply:
No, just add it in the top of the file